org.blackcoin.blackcoin-more.json

```diff
@@ -0,0 +1,73 @@
+{
+  "id": "org.blackcoin.blackcoin-more",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "22.08",
+  "command": "blackmore-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "blackmore",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/ bin/blackmore-qt"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/expatjedi/blk-flatpak-data/releases/download/13.2.0/blackcoin-more-13.2.0-x86_64-linux-gnu.tar.gz",
+          "sha256": "e73c68fd4a35cc7d629f62ccfb265be5b3fa259b162d28c31d15cf6228a8a90a"
+        },
+        {
+          "type": "archive",
+          "only-arches": ["aarch64"],
+          "url": "https://github.com/expatjedi/blk-flatpak-data/releases/download/13.2.0/blackcoin-more-13.2.0-aarch64-linux-gnu.tar.gz",
+          "sha256": "9816fa0772068b55686dda6e1c0eda276942980bfa53ee47495fee6a001311e8"
+        }
+      ]
+    },
```

Comment:
Any reason it uses your personal build instead of building it from source?

Reply:
We already built the binary within our building envoriment so no need to do same task again. I'm sure your system works great but ours too.

> Could you explain why "I am an upstream contributor to the project." is marked as done, but "I own the domain used in the application ID" is not? Just trying to establish how upstream is exactly related to this submission.

I'm one of the developer at blackcoin but not the leading one. If you wish, you may contact lead devs and confirm my role on project.